Lesson 3/main.js

```diff
@@ -0,0 +1,11 @@
+function listChange(obj) {
+    const message = document.getElementById('show_message');
+    const value = obj.value;
+    if(value === "0") {
```

Comment:
```suggestion
    if (value === "0") {
```
dấu nháy đơn
và để ý dấu cách đi

đến buổi thứ bao nhiêu rồi mà vẫn còn như thế này